src/App.tsx

```diff
@@ -14,11 +15,106 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGoods: string[],
+};
 
-export default App;
+export class App extends React.Component<{}, State> {
+  state: State = {
+    selectedGoods: ['Jam'],
+  };
+
+  select = (good: string) => {
+    this.setState(oldState => ({
+      selectedGoods: [...oldState.selectedGoods, good],
+    }));
+  };
+
+  remove = (good: string) => {
+    this.setState(oldState => {
+      oldState.selectedGoods.splice(oldState.selectedGoods.indexOf(good), 1);
+
+      return { selectedGoods: oldState.selectedGoods };
+    });
+  };
+
+  stringOfGoods = (goods: string[]) => {
```

Comment:
```suggestion
  getTitle = (goods: string[]) => {
```
функция должна выполнять действие